code/__DEFINES/language.dm

```diff
@@ -36,6 +36,7 @@
 #define LANGUAGE_VOICECHANGE "voicechange"
 #define LANGUAGE_RADIOKEY "radiokey"
 #define LANGUAGE_BABEL "babel"
+#define LANGUAGE_EMP "emp"
```

Comment:
Vending machines used to be able to be hacked to change languages they talk in. Do we not want to re-add that instead, by giving them their own language holder?